nest-clamav-scanner/src/forms-client/forms-client.service.ts

```diff
@@ -1,65 +1,82 @@
-import { Injectable, Logger } from '@nestjs/common';
-import { ConfigService } from '@nestjs/config';
-import axios, { AxiosError } from 'axios';
+import { Injectable, Logger } from '@nestjs/common'
+import { ConfigService } from '@nestjs/config'
+import axios, { AxiosError, AxiosPromise } from 'axios'
+import {
+  UpdateFileStatusRequestDtoStatusEnum,
+  UpdateFileStatusResponseDto,
+} from 'openapi-clients/forms'
+import ClientsService from '../clients/clients.service'
 
 @Injectable()
 export class FormsClientService {
-  private readonly logger: Logger;
+  private readonly logger: Logger
 
-  constructor(private readonly configService: ConfigService) {
-    this.logger = new Logger('FormsClientService');
+  constructor(
+    private readonly configService: ConfigService,
+    private readonly clientsService: ClientsService,
+  ) {
+    this.logger = new Logger('FormsClientService')
   }
 
   //create function which will check health status of forms client with axios and using forms client url NEST_FORMS_BACKEND
   public async isRunning(): Promise<boolean> {
     try {
-      const url = `${this.configService.get('NEST_FORMS_BACKEND')}/healthcheck`;
+      const url = `${this.configService.get('NEST_FORMS_BACKEND')}/healthcheck`
```

Comment:
Did you forget to replace this?

Reply:
I did replace it, twice in fact, and then I fumbled the commit 🤦‍♂️ Will fix this soon